src/common/components/contactInfoModal/contactInfoModal.tpl.html

```diff
@@ -1,4 +1,4 @@
-<contact-info submitted="$ctrl.submitted" on-submit="$ctrl.onSubmit(success)"></contact-info>
+<contact-info submitted="$ctrl.submitted" on-submit="$ctrl.onSubmit(success)" donor-details="$ctrl.signUpDonorDetails"></contact-info>
```

Comment:
Where is `signUpDonorDetails` defined? Were you also meant to commit changes to the contactInfoModal.component.js

Reply:
It is set in `checkDonorDetails` if there was an error saving the donor details